lib/srv/db/audit_test.go

```diff
@@ -60,13 +66,15 @@ func TestAuditPostgres(t *testing.T) {
 	_, err = psql.Exec(ctx, "select 1").ReadAll()
 	require.NoError(t, err)
 	requireQueryEvent(t, testCtx, libevents.DatabaseSessionQueryCode, "select 1")
+	waitForRecordingEvent(t, recorder, libevents.DatabaseSessionCommandResultCode)
```

Comment:
let's validate that the event has the correct fields. I would also add an error case (status.sucess=false)

Reply:
I had a discussion with @Tener that we want to try out https://testcontainers.com/ for testing against real databases. One use would be the e2e tests for self-hosted. But the other idea is we can have an environment variable in tests like this say `TELEPORT_DB_TESTCONTAINERS=true` which will manually trigger the test containers flow that setup databases in your local containers. Not this PR but we should explore that when get a chance.